crypto/evp/e_aes.c

```diff
@@ -1003,6 +1003,57 @@ typedef struct {
     int tls_aad_len;
 } S390X_AES_GCM_CTX;
 
+typedef struct {
+    union {
+        double align;
+        /*-
+         * Padding is chosen so that ccm.kmac_param.k overlaps with key.k and
+         * ccm.fc with key.k.rounds. Remember that on s390x, an AES_KEY's
+         * rounds field is used to store the function code and that the key
+         * schedule is not stored (if aes hardware support is detected).
+         */
+        struct {
+            unsigned char pad[16];
+            AES_KEY k;
+        } key;
+
+        struct {
+           /*-
+            * KMAC-AES parameter block
+            * (see z/Architecture Principles of Operation SA22-7832-11)
+            */
```

Comment:
Style nit. One space is missing in above commentary block indentation.

On related *side*[!] note I wonder if it would be appropriate to mention earliest version where kmac-aes appeared, instead of latest one that is. I don't know when exactly it was specified, but it goes back to 8... Well, If you don't know, don't bother...

Reply:
Just looked it up - you remember correctly :-)